lua/ts-comments.lua

```diff
@@ -26,6 +26,7 @@ local options = {
     rego = "# %s",
     rescript = "// %s",
     ruby = "# %s",
+    rust = "// %s",
```

Comment:
Rust also has `/// %s` for doc comments

Reply:
I think doc comments should not be covered by this plugin. The `gc` mapping is usually used to toggle comments. The toggled "comments" are usually codes, not natural language sentences. But doc comments are natural language sentences, so it rarely makes sense to toggle doc comments.

I faced some real problems as a result of doc comments being supported. Let's consider the following code snippet, and I want to comment out this function:

```rust
struct Foo {
    /// foo
    foo: ()
}
```

- If I only comment out the `foo: ()` line, I will get the error "expected item after doc comment this doc comment doesn't document anything".
- If I comment out both lines, using the `/// ` doc comment (this is the case for `gcj`/`gck` when the cursor is in the first four columns, inside the indentation, before the comment), then I will get the error "found a documentation comment that doesn't document anything  doc comments must come before what they document, if a comment was intended use `//`".